### PR TITLE
Update 1.25 Makefile to work with CI builds

### PR DESCRIPTION
**Pull request by nskaggs**
## Description of change

We want to allow ourselves the ability to work on 1.25 series as easily as 2.x, if needed. This enables us to define our own separate runs for 1.25.

## QA steps

If CI can build and test 1.25, we're good!

### CODE REVIEW

**Comment by nskaggs** [comment]
!!build!!

**Comment by nskaggs** [comment]
!!build!!

**Comment by nskaggs** [comment]
The current failures are test related; I'd like to land this with the failing tests so we can get a CI run regardless.

File: Makefile [hunk 1]
```diff
@@ -46,18 +53,37 @@ godeps:
...
+check: godeps pre-check
+	go test $(CHECK_ARGS) -test.timeout=$(TEST_TIMEOUT) $(PROJECT)/...
+
+install: godeps go-install
 
 clean:
 	go clean $(PROJECT)/...
 
+go-install:
+	go install -ldflags "-s -w" -v $(PROJECT)/...
```
Comment (Veebers) [inline Makefile]: Why is -ldflags "-s -w" needed?

**Comment by nskaggs** [comment]
$$merge$$

**Pull request merged**